EIPS/eip-4345.md

```diff
@@ -1,41 +1,41 @@
 ---
 eip: 4345
-title: Difficulty Bomb Delay to May 2022
-description: Delays the difficulty bomb to be noticeable in May 2022.
+title: Difficulty Bomb Delay to June 2022
+description: Delays the difficulty bomb to be noticeable in June 2022.
 author: Tim Beiko (@timbeiko), James Hancock (@MadeOfTin)
 discussions-to: https://ethereum-magicians.org/t/eip-4345-difficulty-bomb-delay-to-may-2022/7209
-status: Draft
+status: Review
```

Comment:
Actually this needs a `review-period-end` field.

Reply:
Not for Review, only for Last Call.

Reply:
Yeah we realised after it, I thought it was Last Call as that's the status these go into when they are scheduled for a fork.